dsperse/src/analyzers/runner_analyzer.py

```
@@ -6,202 +6,256 @@
-                })
-
-                if keys_exist:
-                    slice_metadata.update({
-                        "vk_path": vk_path,
-                        "pk_path": pk_path
-                    })
+        return slices
 
-            slices[segment_key] = slice_metadata
+    @staticmethod
+    def _process_slices_per_slice(slices_dir: Path, slices_data_list: list[dict]) -> dict:
```

Ruff (0.14.6)

Unused static method argument: `slices_dir`

(ARG004)